src/components/ProfileInput.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 300,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ProfileInput() {
+  const classes = useStyles();
+  const [grade, setGrade] = React.useState('');
+  const [gradYear, setGradeYear] = React.useState('');
+  const [major, setMajor] = React.useState('');
+  const [gender, setGender] = React.useState('');
+  const [daca, setDaca] = React.useState('');
+
+  const handleGradeChange = (event) => {
+    setGrade(event.target.value);
+  };
```

Comment:
These can be written without the curly braces like this:
```js
const handleGradeChange = (event) => setGrade(event.target.value);
```
You might even shorten `event`:
```js
const handleGradeChange = (e) => setGrade(e.target.value);
```
And might even inline `(e) => setGrade(e.target.value)` into the JSX below

Reply:
Ok nice! I did the inline changes to all of the existing const